tests/test_version_range.py

```diff
@@ -353,6 +355,19 @@ def test_pypi_gitlab_version_range_parse(test_case):
     assert str(result) == test_case["expected_vers"]
 
 
+@pytest.mark.parametrize("test_case", json.load(open("./tests/data/conan_advisory.json")))
+def test_conan_gitlab_version_range_parse(test_case):
+    if test_case["expected_vers"] is None:
+        with pytest.raises(InvalidVersion):
+            Conan1VersionRange.from_native(string=test_case["native"])
+        return
+    result = from_gitlab_native(
+        gitlab_scheme=test_case["scheme"],
```

Comment:
```suggestion
        gitlab_scheme=test_case["gitlab_scheme"],
```

Reply:
Outstanding @TG1999 -- with this and the 2 other sets of changes, all 428 of the test cases pass both Conan-related tests.

Thank you.  🙂 Will commit and push shortly.